test/transforms_test.dart

```diff
@@ -23,20 +23,26 @@ void main() {
     final res = await postgrest
         .from('users')
         .select()
-        .order('catchphrase', ascending: true)
+        .order('status', ascending: true)
         .order('username')
         .execute();
     expect(
-      ((res.data as List)[0] as Map)['username'],
-      'kiwicopple',
-    );
-    expect(
-      ((res.data as List)[2] as Map)['username'],
-      'supabot',
+      (res.data as List).map((row) => (row as Map)['status']),
+      [
+        'ONLINE',
+        'ONLINE',
+        'OFFLINE',
+        'OFFLINE',
+      ],
```

Comment:
For anyone wondering why `ONLINE` comes before `OFFLINE` here like I was, `order by` of enum values in Postgres works not by alphabetical order, but in the order that they were defined by `CREATE TYPE` statement. The enum here was created like this, so `ONLINE` comes first. 

```sql
CREATE TYPE public.user_status AS ENUM ('ONLINE', 'OFFLINE');
```